pom.xml

```diff
@@ -39,7 +39,7 @@
 		<legacyuiVersion>1.3.3</legacyuiVersion>
 		<metadatadeployVersion>1.9.0</metadatadeployVersion>
 		<ugandaemrVersion>3.0.1</ugandaemrVersion>
-		<ugandaemrPocVersion>1.1.0</ugandaemrPocVersion>
+		<ugandaemrPocVersion>1.1.1</ugandaemrPocVersion>
```

Comment:
@ssmusoke can we change this naming convention for uniformity purposes i.e, to lower case all throughout  with all other module naming conventions in the main UgandaEMR module.
NOTE: It has no effect on perfomance.

Reply:
@solemabrothers This is a variable and it does not have an effect on the working of the module so we should not touch it at this time